### DISCUSSION TITLE
[Prettier Plugin] using with astro class:list

### DISCUSSION THREAD

**Discussion by ruridge** [tailwindlabs/tailwindcss#10205] [Help]
Can the prettier plugin be configured to work with the [Astro `class:list` attribute](https://docs.astro.build/en/reference/directives-reference/#classlist)?

**Comment by ony3000** [comment]
Hello! I opened [this PR](https://github.com/tailwindlabs/prettier-plugin-tailwindcss/pull/192) a few days ago, and when this is merged, class sorting will be supported for the `class:list` attribute as well.
